client-code-gen/gen/admin-management-api/model/api-app-constants-user-type1.ts

```diff
@@ -0,0 +1,31 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Forest Access Management - FAM - Admin Management API
+ *  Forest Access Management Admin Management API used by the Forest Access Management application to define admin access to forest applications. 
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: SIBIFSAF@victoria1.gov.bc.ca
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+
+/**
+ * 
+ * @export
+ * @enum {string}
+ */
+
+export const ApiAppConstantsUserType1 = {
+    I: 'I',
+    B: 'B'
+} as const;
```

Comment:
Hi Ian, I'm not sure where this is coming from...

Reply:
This looks curious.
I assume nothing changed other than openapi.json.
I may need to run generation my end to see if I get the same result....
Did you notice if that causes frontend issue or changes needed by this strange generation?

Reply:
no... I reinstalled frontend, and run frontend, looks good...

Reply:
I tried on my end to run generation.
Same to me that it created the strange type.
This is quite strange....

Reply:
I tried several things, including reinstall python .venv but no difference.
It only happens in server/admin_management, not in server/backend for strange reason unknown.
Sorry, I couldn't tell the difference. Since frontend still runs fine (but swagger is having strange type), maybe we ignore this one for now but keep eyes on it for the next time.